.evergreen/config.in.yml

```diff
@@ -509,27 +444,14 @@ functions:
           export AWS_ACCESS_KEY_ID='${AWS_ACCESS_KEY_ID}'
           export AWS_SECRET_ACCESS_KEY='${AWS_SECRET_ACCESS_KEY}'
           EOT
-    - command: shell.exec
+    - command: subprocess.exec
       type: test
       params:
-        working_dir: "src"
-        script: |
-          ${PREPARE_SHELL}
-
-          export PYTHON_BINARY=$([ "Windows_NT" = "$OS" ] && echo "/cygdrive/c/python/python38/python.exe" || echo "/opt/mongodbtoolchain/v3/bin/python3")
-          export PROJECT_DIRECTORY="$(pwd)"
-          export DRIVERS_TOOLS="${DRIVERS_TOOLS}"
-          export NODE_LTS_VERSION='${NODE_LTS_VERSION}'
-          export MONGODB_URI="${MONGODB_URI}"
-          export TEST_SOCKS5_CSFLE="${TEST_SOCKS5_CSFLE}"
-          export SSL="${SSL}"
-
-          # Disable xtrace (just in case it was accidentally set).
-          set +x
-          . ./prepare_client_encryption.sh
-          rm -f ./prepare_client_encryption.sh
-
-          bash ${PROJECT_DIRECTORY}/.evergreen/run-socks5-tests.sh
+        add_expansions_to_env: true
```

Comment:
Half of this block was configuring FLE variables, the other half re-exported expansions. It can all be removed.